### DISCUSSION TITLE
Use box-drawing characters instead of vertical bars as command history separator

### DISCUSSION THREAD

**Discussion by kidonng** [PatrickF1/fzf.fish#167] [Archive] [answered]
I noticed in the screenshot from junegunn/fzf#2512, the vertical bars actually lined up as a whole. However this seems to be specific to fonts/terminals, here is how it looks like on my side:

![image](https://user-images.githubusercontent.com/44045911/120469318-c45f6000-c3d4-11eb-9e25-98c0b9d15b69.png)

Not bad, not ideal either. I propose to use box-drawing characters (`│`) instead of vertical bars (`|`) instead, here's how it may look like:

![image](https://user-images.githubusercontent.com/44045911/120469429-e822a600-c3d4-11eb-9370-af3e71809655.png)

Pros:
- Visually nicer
- Enable users to search for pipes (`|`)

Cons:
- Not easy to type (but I can't think of a reason to have to type it)

**Comment by PatrickF1** [comment, marked as answer]
Yes, I love this idea! And thank you for asking first before opening a PR.

**Reply by PatrickF1** [reply]
Done in https://github.com/PatrickF1/fzf.fish/commit/aaf3652f566c93de370996c52ea7bfceedd799e1
